setup.py

```diff
@@ -28,7 +28,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='0.1.7',
+    version='1.0.0',
```

Comment:
can we add `version.py` and import/use it here as well as in middleware.py file?